src/modules/components/Button.tsx

```diff
@@ -47,10 +47,11 @@ export const Button = React.forwardRef(
         activeBg,
         hoverBg,
         activeHoverBg,
+        pressed,
         ...props 
     } : ButtonProps & StyledButtonProps, ref: React.ForwardedRef<HTMLButtonElement> ) => {
     const componentStyle = { ...DEFAULT_STYLE, ...props}
-    const [pressed, setPressed] = useState<boolean>(false);
+    const [isPressed, setPressed] = useState<boolean>(!!pressed);
```

Comment:
buttons and switches should really be different components with clearly different UI. We'll need to come up with some design or default to some component library (at least for now)